mappings/net/minecraft/level/Level.mapping

```diff
@@ -97,26 +122,40 @@ CLASS net/minecraft/class_18 net/minecraft/level/Level
 		COMMENT This consists of a {@linkplain LevelSource level source} acting as a chunk cache which references another level source which acts as a chunk generator.
 		COMMENT @return the new chunk cache
 	METHOD method_199 getChunk (II)Lnet/minecraft/class_43;
+		ARG 1 x
+			COMMENT The absolute X coordinate
```

Comment:
Can your comment specify that it's chunkX and chunkZ?

Reply:
or, if it's not, that it's block X and block Z. Absolute just means it's not relative